src/runmode-af-packet.c

```diff
@@ -433,21 +464,28 @@ static void *ParseAFPConfig(const char *iface)
     if (ConfGetChildValueWithDefault(if_root, if_default, "xdp-filter-file", &ebpf_file) != 1) {
         aconf->xdp_filter_file = NULL;
     } else {
+#ifdef HAVE_PACKET_XDP
         SCLogInfo("af-packet will use '%s' as XDP filter file",
                   ebpf_file);
+        aconf->ebpf_t_config.mode = AFP_MODE_XDP_BYPASS;
+        aconf->ebpf_t_config.flags |= EBPF_XDP_CODE;
         aconf->xdp_filter_file = ebpf_file;
+        /* TODO FIXME Do we really have a usage of setting XDP and not bypassing ? */
```

Comment:
should this be addressed?

Reply:
I'm removing it. Experience will show if question is coming back.